addons/website_form/static/src/js/website_form_editor_tour.js

```diff
@@ -49,50 +51,6 @@ odoo.define('website_form_editor.tour', function(require) {
             content:  "Click on Save",
             trigger:  ".modal-footer #modal-save"
         },
-        // Add the subject field
-        {
-            content:  "Click on Form snippet",
-            trigger:  ".s_website_form[data-model_name]"
-        },
-        {
-            content:  "Click on Customize",
-            trigger:  ".oe_overlay_options [title='Customize']"
-        },
-        {
-            content:  "Click on Add a model field",
-            trigger:  ".oe_overlay_options .dropdown-item[data-website_form_field_modal]"
-        },
-        {
-            content:  "Select the subject field",
-            trigger:  "select[name='field_selection']",
-            run:      "text subject"
```

Comment:
I think you removed that part of the test since subject field is now by default on the form but could you add another field and keep those steps so the test keeps checking the `required` and `name` part.

Reply:
Done!